### PR TITLE
Create Wiki article about design resources

### PR DESCRIPTION
**Pull request by TheHighPriestess**
Wiki article with 10 resources (mostly free) where front-end developers can learn the basics of design. 

I wrote it on a whim and I'd like to offer it to the community.  I think it would be especially useful for people interested doing freelance work.

### CODE REVIEW

**Comment by koustuvsinha** [comment]
@TheHighPriestess thank you for your article! however, please change the file to have a **.md** extension. Also, capitalize the file like this : 

```diff
- ten-must-have-design-resources-for-front-end-developers
+ Ten-Must-Have-Design-Resources-For-Front-End-Developers.md
```

do not forget the .md at the end!

File: ten-must-have-design-resources-for-front-end-developers [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+wouldn’t have read this far. Well, how are you going to be a great, or even a good,
+designer if you’re clueless about your users? You gotta think about those users.
+They matter because you are building websites that people are going to use. Be
+kind to them. Learn something about user psychology.
+ 
+## 5. [Material Palette](https://www.materialpalette.com/)
+
+An attractive color palette alone can go a long way toward making a design look
+professional. So, if you’ve got through the basic design stuff I mentioned above but
+still aren’t sure about how to apply color theory bookmark this site now and start
```
Comment (theflametrooper) [inline ten-must-have-design-resources-for-front-end-developers]: ~~color theory bookmark~~ color theory, bookmark

**Comment by Rafase282** [comment]
Right now it is ten, but in the future we could add or remove, please rename the file to something like 
`Design-Resources.md`

**Comment by Rafase282** [comment]
Could use a TOC but other than that 

# LGTM

File: Design-Resources-For-Front-End-Developers.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
+# Design Resources for Front-End Developers
+
+Are you a freelance front-end web developer or are you thinking about becoming
+one? Maybe you’ve been to a coding bootcamp or graduated from an online
```
Comment (alayek) [inline Design-Resources-For-Front-End-Developers.md]: > Maybe you’ve been to a

Change it to 

> Maybe you've been to a

File: Design-Resources-For-Front-End-Developers.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+front-end development course. What happens next?
+
+You get your first customer. It might just be a friend who needs a personal website.
+It doesn’t matter. You don’t want it to look amateurish. You want it to look good. You
+want it to look professional.
+
+Here’s the thing…
```
Comment (alayek) [inline Design-Resources-For-Front-End-Developers.md]: Same as [above](#r63223221)

File: Design-Resources-For-Front-End-Developers.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+Good design can mean the difference between a website that looks ‘homemade’
```
Comment (alayek) [inline Design-Resources-For-Front-End-Developers.md]: > ‘homemade’

Same as [above](#r63223221)

File: Design-Resources-For-Front-End-Developers.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+and one that looks professional. If you’re going to make money at freelance web
```
Comment (alayek) [inline Design-Resources-For-Front-End-Developers.md]: > you’re

Same as [above](#r63223221)

**Comment by Rafase282** [comment]
I'll make the changes.

**Pull request merged**